cmake/GenerateBuiltins.cmake

```diff
@@ -158,8 +158,16 @@ function(builtin_to_cpp bit os_name arch supported_archs supported_oses resultFi
     endif()
 
     # OS to arch constrains
-    if (${os_name} STREQUAL "windows" AND ${arch} STREQUAL "arm")
-        set(SKIP ON)
+    if (${os_name} STREQUAL "windows")
```

Comment:
No need to set `target_arch` to `arm64`. `arm64` is Apple specific name for `aarch64`.

I suggest just adding to the original condition `AND "${bit}" STREQUAL "32"` This should be sufficient - this will skip ARM 32bit only.